test/torchaudio_unittest/transforms/batch_consistency_test.py

```diff
@@ -58,6 +58,24 @@ def test_batch_InverseMelScale(self):
         # exactly same result. For this reason, tolerance is very relaxed here.
         self.assert_batch_consistency(transform, mel_spec, atol=1.0, rtol=1e-5)
 
+    def test_batch_BarkScale(self):
+        specgram = torch.randn(3, 2, 201, 256)
+
+        atol = 1e-4 if os.name == "nt" else 1e-6
```

Comment:
do you know why these tolerances are increased from the working melscale ones above?